### DISCUSSION TITLE
Individual image precision

### DISCUSSION THREAD

**Discussion by MABatin** [open-mmlab/mmdetection#7986] [Q&A]
Hi, I was wondering how I could check the precision metric for individual test images of a dataset with COCO format. Is there a hook already implemented for this or do I need to write my own script?

**Comment by ZwwWayne** [comment]
For nor we do not support that. You can take a look at the analysis_results.py which support to evaluate per-image AP https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/analyze_results.py.
